### ISSUE TITLE
Question about formatting list of sequence types (for ALF web tool)

### ISSUE THREAD

**Issue by bioinfoMMS** [DessimozLab/ALF#1]
Not sure if this is the bast place to ask this, but I am trying to run the web version of ALF and I would like to assign sequence types to different parts of my custom root genome. The web tool has a section asking for the user to "select a file with a list, assigning a specific sequence type to each sequence in the root genome (only works for custom root genome)". Could you let me know how to format this list within in the file?

**Comment by alpae** [comment]
Hi @bioinfoMMS 
not very familiar with the ALF codebase, but as far as I can see it is just a text file that defines a list with numeric values corresponding to the individual types. so if you have a root genome with 5 genes and define 3 different sequence types, you would specify it as 
`[1,2,3,1,2]`

**Comment by bioinfoMMS** [comment]
Thank for you response!
Would the 5 genes in your example be separated in the root genome file i.e the genome file would contain multiple (5) fasta sequences? if not, how are the locations of the genes in the genome determined?

**Comment by alpae** [comment]
yes, exactly. The root genome needs to have explicit fasta sequences, one per gene. The absolute location is not modeled in ALF, but rather the relative order. The order of the sequence type array corresponds to the order of the genes provided in the root genome fasta file and their length must match.

**Comment by bioinfoMMS** [comment]
I see, thank you so much for your help!! Have a great day!
